app/error_handler.py

```diff
@@ -0,0 +1,60 @@
+"""Error handler module.
+
+
+"""
+import logging
+
+from app.configuration import Configuration
+from app.mqtt_writer import MqttWriter
+from app.exceptions import InvalidArgumentError
+import paho.mqtt.client as mqtt
+
+
+class ErrorHandler:
+    class Category:
+        MODBUS_ERROR = "ModbusError"
+        MQTT_ERROR = "MQTTError"
+        INVALID_MESSAGE = "InvalidMessage"
+        UNKNOWN_COMMAND = "UnknownCommand"
+
+    @classmethod
+    def from_config(cls, config: Configuration, mqtt_client: mqtt.Client = None):
```

Comment:
Is this needed?

Reply:
BTW there is a name for this
https://lostechies.com/chadmyers/2009/07/14/the-usual-result-of-poor-man-s-dependency-injection/
It's often not recommended.

The term though has two meanings. One is not having a container and do all the instantiation manually. The other is partially injected classes. I'm talking about the latter.